### ISSUE TITLE
When installing an Astarte trigger, the Mustache template is not included in the definition

### ISSUE THREAD

**Issue by davidebriani** [astarte-platform/astartectl#263]
Trying to install an Astarte trigger that specify that a Mustache template should be used instead of the default JSON payload, the trigger is installed without the Mustache template, i.e. the default JSON payload will be used for the trigger action.

The test was made trying to install a trigger like the following with `astartectl realm-management triggers sync`

```json
{
    "name": "alert-when-temperature-is-high",
    "action": {
        "http_url": "https://domain.com",
        "http_method": "post",
        "http_static_headers": {
            "Accept": "application/json",
            "Content-Type": "application/json"
        },
        "ignore_ssl_errors": true,
        "template_type": "mustache",
        "template": "{\n  \"title\": \"High temperature\",\n  \"message\": \"The temperature measured by the device {{ device_id }} is high, its current value is {{ value }} °C.\",\n  \"recipients\": [\"some.recipient@domain.com\"]\n}"
    },
    "simple_triggers": [
        {
            "type": "data_trigger",
            "on": "incoming_data",
            "interface_name": "some.interface.Interface",
            "interface_major": 0,
            "match_path": "/temperature",
            "value_match_operator": ">",
            "known_value": 35
        }
    ]
}
```

**Comment by eddbbt** [comment]
astarte-go was missing support for template and template_type, https://github.com/astarte-platform/astarte-go/pull/64 fixes that

**Comment by Annopaolo** [comment]
Closed by #267
